compiler-test/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'software.amazon.lib.jvm'
+}
+
+dependencies {
+    implementation project(':compiler-utils')
+    implementation project(':runtime')
+    implementation libs.ksp.api
```

Comment:
Some of the dependencies, like KSP and the compile testing, must be an `api` dependency.